hermes-api/src/main/java/pl/allegro/tech/hermes/api/SentMessageTrace.java

```diff
@@ -142,6 +142,15 @@ public boolean equals(Object obj) {
             return false;
         }
         final SentMessageTrace other = (SentMessageTrace) obj;
-        return Objects.equals(this.messageId, other.messageId) && Objects.equals(this.subscription, other.subscription);
+        return Objects.equals(this.messageId, other.messageId)
```

Comment:
ok, I've added timestamp and status fields only as they define what trace exactly it is.